setup.cfg

```diff
@@ -9,5 +9,5 @@ max-line-length = 120
 
 [flake8]
 max-line-length = 120
-ignore = F401,E402,F403,W503,W504
+ignore = F401,E402,F403,W503,W504,F821
```

Comment:
Interesting, in the future we might want to remove this F821 and instead fix the annotations